### PR TITLE
Section about nullable dtypes

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
A first dump of some content: a single text not yet split up into work plan / milestones

### CODE REVIEW

File: proposal.md [hunk 1]
```diff
@@ -55,6 +55,54 @@ the end of September.
...
+
+- Provide a mechanism to easily opt-in to use the nullable data types across the board.
+- General improvements to the Extension Array interface:
+  - Enable using extension arrays for the Index
+  - Better support and customization of construction and casting operations (`astype()`)
+  - Remaining numerical operations (e.g. `round`, `count`, cumulative methods, numpy protocols, ...)
+- Specific improvements to the masked array implementation and algorithm support of the nullable data types:
+  - Investigate ways to optimize the storage (optional masks, bitmasks)
+  - Support masked arrays directly in more algorithms
+- Expand nullable support to new data types (pending discussions on which data types to support).
```
Comment (jorisvandenbossche) [inline proposal.md:97]: Not sure how detailed this section needs to go?

File: proposal.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ urlcolor: cyan
 numbersections: true
 autoidentifiers: true
 authors:
-  - Joris van den Bossche
+  - Joris Van den Bossche
```
Comment (TomAugspurger) [inline proposal.md:9]: Sorry :)

File: proposal.md (hunk 1 above)
Comment (TomAugspurger) [reply proposal.md:97]: I think this is sufficient.

**Comment by TomAugspurger** [comment]
FYI, I forgot to include the prompts:

Work Plan:

> Provide a description of the proposed work the applicants are requesting funding for, including resources the applicants will provide that are not part of the requested funding. For software development related work (e.g., engineering, product design, user research), specify how the work fits into the existing software project roadmap. For community outreach related activities (e.g., sprints, training), specify how these activities will be organized, the target audience, and expected outcomes (maximum of 750 words)


Milestones and Deliverables:

> List expected milestones and deliverables, and their expected timeline. Be specific and include (where possible) any goals for metrics the software project(s) are expected to reach upon completion of the grant (maximum of 500 words)

**Comment by jorisvandenbossche** [comment]
(BTW, feel free to some strong editing, as you are much better at fluent english writing as me ;-) Will look a bit more based on those descriptions of the sections tomorrow)

**Pull request merged**